app/javascript/flavours/glitch/styles/rtl.scss

```diff
@@ -1,131 +1,43 @@
 body.rtl {
   direction: rtl;
 
-  .column-header > button {
-    text-align: right;
-    padding-left: 0;
-    padding-right: 15px;
-  }
-
-  .radio-button__input {
-    margin-right: 0;
-    margin-left: 10px;
-  }
-
-  .display-name {
-    text-align: right;
-  }
```

Comment:
This causes a layout change because of the `bdi` tag.

That being said, I notice now this just swaps the behavior that currently exists with right-to-left names in a left-to-right interface, so addressing this is probably out of scope for this PR.

(The misalignment is something I failed to notice upstream but is there too, so out of scope for this PR as well)

## Before

![image](https://user-images.githubusercontent.com/384364/230865754-caaee89e-0312-4120-9465-e83eae148f3e.png)

## After

![image](https://user-images.githubusercontent.com/384364/230866040-fc78d89a-b87c-4d49-af65-60261c663f07.png)